CopyQ.py

```diff
@@ -0,0 +1,129 @@
+""" CopyQ Clipboard Management """
+
+import subprocess
+from albertv0 import *
+from shutil import which
+
+__iid__ = "PythonInterface/v0.1"
+__prettyname__ = "CopyQ"
+__version__ = "1.0"
+__trigger__ = "cq "
+__author__ = "Lukas Holecek"
```

Comment:
Who is Lukas? I didnt want to remove your name :D I just wanted to keep credits for former contributors. Just add your name to the string, separated by comma.

Reply:
Ah, now I see. Lukas is the author of copyq. The __autor__ variable is intended to specify the author[s] of the extension, i.e. you and BarbUk .

Reply:
Ah, I now understand :P